### PR TITLE
MOB-2220 can not comment activity

### PR DESCRIPTION
**Pull request by nttuyen**
- The cause of this bug: it's a regression after feature edit activity/comment, it stop js event bubble up to open event view , and open the mobile editor there
- Fix JS bug to have comment editor shown on mobile

### CODE REVIEW

**Comment by trongtt** [comment]
@nttuyen please add description of the problem in PR

**Comment by nttuyen** [comment]
> @nttuyen please add description to explain the problem in PR

@trongtt , i'm done adding problem description

File: webapp/resources/src/main/webapp/javascript/eXo/social/webui/UIActivity.js [hunk 1]
```diff
@@ -249,7 +249,6 @@
       var commentLinkEl = $("[data-activity='" + activityId + "']");
       if (commentLinkEl.length > 0) {
         commentLinkEl.off('click').on('click', function (evt) {
-          evt.stopPropagation();
```
Comment (trongtt) [inline webapp/resources/src/main/webapp/javascript/eXo/social/webui/UIActivity.js]: I think this was added on purpose. Can you double check carefully if this is really useless ?

Comment (nttuyen) [reply webapp/resources/src/main/webapp/javascript/eXo/social/webui/UIActivity.js]: ok i'll re-check to see if there is regression

Comment (nttuyen) [reply webapp/resources/src/main/webapp/javascript/eXo/social/webui/UIActivity.js]: @trongtt , i've just re-check, i don't find any regression, i've checked feature: create new, edit , delete, and like on both activity and comment

**Pull request merged**